contrib/snapcraft/snap/snapcraft.yaml

```diff
@@ -13,14 +13,15 @@ base: core18
 
 parts:
   podman:
-    plugin: go
-    source: https://github.com/containers/libpod/archive/v0.11.1.1.tar.gz
-    go-importpath: github.com/containers/libpod
+    plugin: make
+    source: ../..
+    build-snaps:
+     - go/1.13/stable
     build-packages:
      # https://github.com/containers/libpod/blob/master/install.md#build-and-run-dependencies
      - btrfs-tools
      - git
-     - golang-go
+     #- golang-go
```

Comment:
if it is not needed please drop it instead of commenting it out

Reply:
Dropped. It doesn't help failing `gating` still.